assets/stylesheets/main.css.map

```diff
@@ -0,0 +1 @@
I,YV1BU,SU4BZ,MZjCY,QYkCZ,eV3BoB,QSGtB,WACE,YTTsB,mDSWxB,YACE,YTduB,yFSgBzB,OACE,YTnBkB,mEWClB,iBACE,OXiBS,MWhBT,MXgBS,MWXX,yCAGE,KbVG,0BcCH,oBAIA,6CACE,aCPN,WACE,uFAEA,eACA,gBACA,yBACA,mCACA,4BACA,uBACA,oBACA,oBACA,kBAGF,4BACE,gBACA,cAEF,4BACE,oBACA,mBAEF,iBACE,gBACA,aAGF,sBACE,eAIF,oBACE,oIAEA,0FAEA,yBAIF,iBACE,iIAEA,uFAEA,yBAGF,qDACE,4BACA,qBCnDF,eACE,UAEA,iBACE,MhBJI,KgBON,kBACE,oBACA,mBACA,sBAEA,yBACE,Ydaa,KcRb,iCACE,+BACA,oCACA,qCACA,SACA,WACA,SACA,oBACA,kBACA,kBAMJ,sEAGE,qBAEA,+EACE,0BCvCR,kBACE,kCACA,MjBGK,QiBDL,yEAGE,kBjBKE,QiBJF,MjBEG,QiBDH,qBAEA,+EACE,MjBFC,QiBGD,qBAIJ,gDAEE,iBjBXY,QkBRhB,aACE,uBACA,OACA,oBACA,kBACA,YACA,+BACA,YAGF,mBACE,gBACA,OAEA,kBACA,MACA,+BCdA,uEACE,2CAIJ,sBACE,iBnBEK,QmBDL,aACA,kBACA,QACA,OACA,YAIA,6BACE,iBnBbG,QmBcH,gBACA,gBACA,aAEA,+BACE,oBACA,iBAKN,uBACE,iBnB1BK,QmB2BL,aACA,kBACA,UACA,YACA,YCpCF,sBACE,yBACA,kBACA,8BACA,uBACA,uBACA,gBACA,UCPF,KACE,WACA,gBAGF,iDTUE,iBACA,kBACA,UVaU,MvFdV,mC0GXF,iDTmBI,aVOgB,OUNhB,cVMgB,QmBpBpB,kBTIE,iBACA,kBACA,UVaU,MUXR,aVamB,OUZnB,cVYmB,OvFhBrB,mC0GLF,kBTaI,aVOgB,OUNhB,cVMgB,QmBhBpB,4BACE,aACA,yBAIA,kCTNA,iBACA,kBACA,UVaU,MvFdV,mC0GKA,kCTGE,aVOgB,OUNhB,cVMgB,QmBJlB,0CACE,kBAGF,uEACE,eAGF,iCACE,QAGF,kHAEE,WACA,gBACA,sBAIJ,YACE,sBACA,cAEA,wBACE,aACA,yBACA,2DAGF,cACE,MrBrDG,QqByDP,kBACE,cnB9Ca,kBmBiDf,YACE,cACA,cAEA,uBACE,aACA,eACA,mBACA,8B1G1DF,mC0GgEE,iCACE,MnBlDc,OmBwDd,mGAEE,kBACA,UAEA,0OAEE,oCAEA,sQACE,aAQR,iCACE,oBACA,sBAKN,iBACE,2BACA,oF1G9EA,mC0GmFA,sBACE,eACA,gBAIA,oBACE,cACA,iBAIJ,qBACE,WAEA,2DACE,eAIJ,iBACE,cACA,eAKF,gBACE,8BCnJJ,KACE,WtBKW,QsBJX,kBACA,uBACA,iBACA,gBACA,mBACA,4BAEA,aACE,WACA,iDACA,qBACA,kBACA,UACA,WACA,YACA,WCjBJ,yBACE,gBACA,oBACA,mBAGE,0CACE,+FAIF,uCACE,wFAIF,yCACE,6FAKJ,wCACE,qBACA,kBAEA,+CACE,WACA,qBACA,6DACA,iBACA,cACA,aACA,kBACA,UACA,WAIJ,6DACE,iB5GJF,mC4GWI,6DACE,cAGF,8CACE,iBCnDN,sCACE,+FAEA,wBAGF,yCACE,WtBSW,kBsBRX,0FAEA,wBAGF,iCACE,a7GqBF,mC6GfE,sCACE,gBACA,mBACA,UAGF,yCACE,WtBXS,kBsBYT,gBACA,mBACA,UAGF,iCACE,cAEA,iDACE,yGAEA,aAGF,oDACE,mGAEA,aACA,iBC9CN,oCACE,8GAEA,wBAGF,oCACE,aACA,eAEA,sCboBA,YVvBa,QU2Bf,MZjCY,QYkCZ,eV3BoB,QuBIhB,aACA,YAEA,kDACE,qFAEA,sBAGF,iDACE,uFAEA,YvBRO,kBuBSP,qBACA,sBAKN,kCACE,aAIJ,sCACE,aACA,mB9GHA,mC8GQE,6BACE,aACA,gBAGF,oCACE,gBACA,mBACA,aAGF,oCACE,cACA,kBAGE,iDACE,iBACA,cACA,kBACA,8BAKN,kCACE,cAEA,+CACE,wHAEA,YACA,gBAKN,sCACE,mBAEA,+CACE,kBAGF,4CACE,gBCxFJ,iCACE,aACA,kBAKF,yBACE,0BACA,gCACA,4BAEF,2BACE,8EAEF,2BACE,oFACA,6BAEF,2BACE,4EAEF,2BACE,a/GYF,mC+GNE,yBACE,gBACA,UAGF,2BACE,cACA,YACA,2BACA,4BAEA,sCACE,8EAEF,sCACE,oFAEF,sCACE,6EChDR,eACE,yBACA,WACA,sEACA,YACA,SACA,kBACA,qCAEA,+DAGE,WAGF,6BACE,cACA,YACA,gBACA,WAGF,mCACE,mBACA,mBAGF,gCACE,iBACA,kBAGF,sBACE,aAGF,sBACE,eAGF,+BACE,mBACA,gBAGF,yBACE,yBACE,eACA,MAGF,mCACE,sBAGF,gCACE,YAIJ,yBACE,yDAEE,aC9DJ,cACE,cAEA,6CAEE,iCAGF,iBACE,UACA,gBACA,qBAEA,sBACE,kBAKN,4BACE,mBAIJ,aACE,mBAGE,uBACE,WACA,cACA,YACA,WACA,mBACA,gCACA,wBACA,4BAKF,8BACE,0CAKF,8BACE,0CAKF,8BACE,0CAKF,8BACE,0CAKF,8BACE,0CAIJ,mBACE,qBAEA,6BACE,WACA,qBACA,WACA,UACA,kBACA,sBACA,kBACA","file":"main.css"}
```

Comment:
I see changes to this file after running `make build` in a clean install. Should those be committed? Was it better that this file just be `.gitignore`'d?